src/include/sof/trace/trace-verbose-private.h

```diff
@@ -0,0 +1,559 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Artur Kloniecki <arturx.kloniecki@linux.intel.com>
+ */
+
+#ifndef __SOF_TRACE_TRACE_VERBOSE_PRIVATE_H__
+#define __SOF_TRACE_TRACE_VERBOSE_PRIVATE_H__
+
+#include <config.h>
+
+#define unused(...) do { UNUSED(__VA_ARGS__); } while (0)
```

Comment:
I feel this one can be moved to a public header or UNUSED itself can be updated.